catalog/rocks/entry_catalog.go

```diff
@@ -111,15 +110,9 @@ func NewEntryCatalog(cfg *config.Config, db db.Database) (*EntryCatalog, error)
 
 	pebbleSSTableCache := pebble.NewCache(tierFSParams.PebbleSSTableCacheSizeBytes)
```

Comment:
Why share the same cache instance between the 2?
I would the same proportions (not percents ;) ) of the disk space and create 2 matching pebble caches.
Like we don't want the 2 competing for disk space, we should also avoid competition for memory.

Reply:
I'm not sure it makes sense even for the local tier.  But separate caching _always_ gives *worse* throughput.  Unlike the disk local tier, every block in the pebble memory cache should cost exactly the same to fetch: it's just a read from a disk.  (Well, if you're using disks made of spinning rust you _might_ see a difference, but who does that?)

Given that this PR is mostly a rollback of an expensive premature optimization, I am hesitant to add more options to this.

*NOTE TO SELF:* Changes (removes) config parameters, so requires a doc change and _should not be pushed today_, let rocks go... :-)

Reply:
Not sure that's true in this case since you will always read the the metarange entry before reading the range entry, so cleaning the metarange entry has potentially a higher cost.

However I'm far from having numbers that support that and indeed it's more of an optimization -> resolving.